### DISCUSSION TITLE
can't run freac 1.1.7-x64 on macOS Catalina 10.15.7

### DISCUSSION THREAD

**Discussion by TimFH** [enzo1982/freac#468] [Help & Support]
Decided to give freebsd version of freac a try, but my system doesn't have "pkg" and home brew doesn't know anything about it, so I downloaded the tar file, extracted it, and on typing ./freac & ./freeaccmd got "-bash: ./freaccmd: cannot execute binary file". The command "file freac" reports:

freac: ELF 64-bit LSB executable, x86-64, version 1 (FreeBSD), dynamically linked, interpreter /libexec/ld-elf.so.1, for FreeBSD 13.1, FreeBSD-style, stripped

I do have libcurl installed, as shown by "ls /usr/lib/libcurl*" outputting:

/usr/lib/libcurl.3.dylib@	/usr/lib/libcurl.4.dylib*	/usr/lib/libcurl.dylib@

So what's wrong (and how can I fix it)?

**Comment by enzo1982** [comment]
I'm confused. The topic says macOS Catalina, but in the post you're talking about the FreeBSD version of fre:ac. Are you trying to run the FreeBSD version of fre:ac on macOS? Why would you expect that to work? Of course macOS cannot run FreeBSD binaries.

Please get the macOS version of fre:ac from here: https://github.com/enzo1982/freac/releases/tag/v1.1.7
